Packs/ThreatExchange/ReleaseNotes/2_0_12.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### ThreatExchange v2
+
```

Comment:
You also added the *Share Level Type* integration parameter - worth mentioning here.